src/one_dragon/base/operation/one_dragon_context.py

```
@@ -297,11 +297,55 @@ def init(self) -> None:
+                        ["git", "config", "core.hooksPath", ".githooks"],
+                        cwd=str(project_root)
+                    )
+                    log.info("Git hooks 路径已成功配置为 .githooks")
+                    
+                    # 赋予执行权限
+                    pre_push_file = githooks_dir / "pre-push"
+                    if pre_push_file.exists():
+                        try:
+                            pre_push_file.chmod(0o755)
+                        except Exception:
+                            pass
```

Ruff (0.15.17)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)